src/containers/SignUp.jsx

```diff
@@ -43,68 +44,55 @@ class SignUp extends Component {
-              />
-
-              <Field
-                label="Enter a password (8 or more characters)"
-                name="password"
-                type="password"
-                component={TextInput}
-              />
-
-              <Field
-                label="Confirm Password"
-                name="confirm_password"
-                type="password"
-                component={TextInput}
-              />
-
-              <Button variant="outlined" color="primary" disabled={!valid} type="submit">
-                Sign up
-              </Button>
-
-              <Button
-                color="primary"
-                type="button"
-                onClick={() => push('/login')}
-                style={{ marginLeft: '1rem' }}
-              >
-                Back
-              </Button>
-            </form>
-
-            <Dialog onClose={this.handleClose} open={open}>
-              <DialogContent>
-                <DialogContentText id="alert-dialog-description">
-                  To complete the sign up process, check your inbox for a link to verify your email.
-                </DialogContentText>
-              </DialogContent>
-            </Dialog>
-
-            {errorMessage ? (
-              <FormHelperText error>
-                {errorMessage}
-              </FormHelperText>
-            ) : null}
-          </div>
-        </div>
-      </div>
+      <PageContainer>
+        <SectionHeaderContainer>
+          <SectionHeader>Sign Up</SectionHeader>
+          <SectionText>Please fill out the form below to create an account.</SectionText>
+        </SectionHeaderContainer>
+        <form onSubmit={handleSubmit(this.onSubmit.bind(this))}>
+          <Field
+            label="Email"
+            name="email"
+            type="text"
+            placeholder="Email"
+            component={TextInput}
+          />
+
+          <Field
+            label="Enter a password (8 or more characters)"
+            name="password"
+            type="password"
+            placeholder="Password"
+            component={TextInput}
+          />
+
+          <Field
+            label="Confirm Password"
+            name="confirm_password"
+            type="password"
+            placeholder="Confirm Password"
+            component={TextInput}
+          />
+
+          <StyledButton filled disabled={!valid} type="submit">
+            Sign up
+          </StyledButton>
+
+          <StyledButton type="button" onClick={() => push('/login')}>
+            Back
+          </StyledButton>
+        </form>
+
+        <Dialog onClose={this.handleClose} open={open}>
+          <DialogContent>
+            <DialogContentText id="alert-dialog-description">
+              To complete the sign up process, check your inbox for a link to verify your email.
+            </DialogContentText>
+          </DialogContent>
+        </Dialog>
+
+        {errorMessage && <ErrorText>{errorMessage}</ErrorText>}
```

Comment:
this might be nitpicking (and not that necessary atm), but you may want to hide the non-current errors. so in this case when an invalid email is inputted it should hide the other error message 

![image](https://user-images.githubusercontent.com/7741746/68524928-4fbc5200-0281-11ea-9b5a-66d515ff5353.png)